### PR TITLE
arch/risc-v: Enable FPU for K210

### PR DESCRIPTION
**Pull request by no1wudi**
## Summary

K210 support rv64gc ISA, now we enable F/D extension for it.

Note: QEMU for K210 don't support FPU now.

## Impact
K210
## Testing

maix-bit

### CODE REVIEW

**Review by pkarashchenko — APPROVED** [review]

File: boards/risc-v/k210/maix-bit/configs/kostest/defconfig [hunk 1]
```diff
@@ -45,4 +45,5 @@ CONFIG_START_MONTH=2
 CONFIG_START_YEAR=2020
 CONFIG_TASK_NAME_SIZE=20
 CONFIG_TESTING_OSTEST=y
+CONFIG_TESTING_OSTEST_FPUTESTDISABLE=y
```
Comment (xiaoxiang781216) [inline boards/risc-v/k210/maix-bit/configs/kostest/defconfig:48]: why disable fpu test?

Comment (no1wudi) [reply boards/risc-v/k210/maix-bit/configs/kostest/defconfig:48]: Need addtion board level  arch_getfpu implementation etc.

Comment (xiaoxiang781216) [reply boards/risc-v/k210/maix-bit/configs/kostest/defconfig:48]: It doesn't need anymore after https://github.com/apache/incubator-nuttx/pull/6080.

Comment (no1wudi) [reply boards/risc-v/k210/maix-bit/configs/kostest/defconfig]: Got it

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**